Makefile.install

```diff
@@ -101,12 +101,16 @@ INSTALLCMI = $(sort \
 	$(foreach lib,$(CORECMA), $(addsuffix .cmi,$($(lib:.cma=_MLLIB_DEPENDENCIES))))) \
 	$(PLUGINS:.cmo=.cmi)
 
+INSTALLCMX = $(sort $(filter-out checker/% ide/% tools/% dev/% configure.cmx, $(MLFILES:.ml=.cmx)))
```

Comment:
Is it right not to install e.g. `ide/ide_slave.cmx`?

Reply:
In theory this is the responsibility of `Makefile.ide:install-ide-devfiles`, given the current code organization, so the above code is consistent, and I indeed prefer to keep the install target separate.

The "IDE" needs additional fixes in order to work as a development base smoothly in both `local` and `global` install.

Reply:
Ok, it's always a bit strange to see `Makefile.ide` in charge of coq files (as opposed to CoqIDE ones), but if that is how it is...

Reply:
`ide_slave` is not a Coq file.

Reply:
In fact it has been fully private for some time, there is this flag in coqtop but that's all.

Reply:
> ide_slave is not a Coq file.

Oh yes, I was confused by the fact that it is dynamically linked with `coqtop`, but indeed that's a CoqIDE file.

Reply:
Out of curiosity, are `.cmt` files installed somehow?

Reply:
`.cmt` files are not installed, and I am not sure they are even needed anymore? `coqtop` does indeed load different plugins. The idea is to share parsing of arguments, but I am not sure the tradeoff is worth it, as code becomes more modular.

Reply:
I use merlin so I need `.cmt` files. What do you use?

Reply:
Surprisingly it works without them sometimes, I am not sure what is going on.